examples/workflow/1_create_model.ipynb

```diff
@@ -0,0 +1,349 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you are happy with the default settings generated by the ``StateTransitionManager``, just start with solving directly!"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. note::\n",
+    "    This step takes about 30 sec on an Intel(R) Core(TM) i7-6820HQ CPU\n",
+    "    @ 2.70GHz running multi-threaded"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "solutions, violated_rules = tbd_manager.find_solutions(\n",
+    "    graph_interaction_settings_groups)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pycompwa.expertsystem.topology.graph import get_intermediate_state_edges\n",
+    "\n",
+    "def print_intermediate_states(solutions):\n",
+    "    \"\"\"Just a little function to print the intermediate states.\"\"\"\n",
+    "    print(\"intermediate states:\")\n",
+    "    intermediate_states = set()\n",
+    "    for g in solutions:\n",
+    "        edge_id = get_intermediate_state_edges(g)[0]\n",
+    "        intermediate_states.add(g.edge_props[edge_id]['@Name'])\n",
+    "    print(intermediate_states)\n",
+    "\n",
+    "print(\"found\", len(solutions), \"solutions!\")\n",
+    "print_intermediate_states(solutions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we have a lot of solutions that are actually heavily suppressed (they involve two weak decays).\n",
+    "\n",
+    "In general, you can modify the dictionary returned by `prepare_graphs()` directly, but the STM also comes with functionality to globally choose the allowed interaction types.\n",
+    "\n",
+    "So, go ahead and **disable** the **EM** and **weak** interaction:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pycompwa.expertsystem.ui.system_control import InteractionTypes\n",
+    "\n",
+    "tbd_manager.set_allowed_interaction_types([InteractionTypes.Strong])\n",
+    "graph_interaction_settings_groups = tbd_manager.prepare_graphs()\n",
+    "solutions, violated_rules = tbd_manager.find_solutions(\n",
+    "    graph_interaction_settings_groups)\n",
+    "\n",
+    "print(\"found\", len(solutions), \"solutions!\")\n",
+    "print_intermediate_states(solutions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Huh, what happened here? Actually, since a **$\\gamma$ particle** appears in one of the interaction nodes, the expert system knows that this node **must involve EM interactions**! Because the node can be an effective interaction, the weak interaction cannot be excluded, as it contains only a subset of conservation laws.\n",
```

Comment:
The gamma character is not rendered correctly in the html

Reply:
https://github.com/ComPWA/pycompwa/pull/74/commits/9da9d510d35c6c3753ea985f2b066b3c56372e4d

Reply:
Latest rendering [here](https://redeboer.github.io/testpages/usage/workflow/1_create_model.html#1.3.-Find-solutions) (temporary link)